data/dataset/scylladb_example_test_dataset.yml

```diff
@@ -0,0 +1,78 @@
+dataset:
+  - fides_key: scylladb_example_test_dataset
+    data_categories: []
+    description: ScyllaDB dataset containing a users table and user_activity table.
+    name: Example ScyllaDB dataset
```

Comment:
example scylla dataset -- some of the data categories are. bit wonky, I auto-generated the dataset using the D & D feature so categories were added by the classifier 😅